modules/exploits/windows/local/ms14_070_tcpip_ioctl.rb

```diff
@@ -0,0 +1,177 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'msf/core/exploit/local/windows_kernel'
+require 'rex'
+
+class Metasploit3 < Msf::Exploit::Local
+  Rank = AverageRanking
+
+  include Msf::Exploit::Local::WindowsKernel
+  include Msf::Post::File
+  include Msf::Post::Windows::FileInfo
+  include Msf::Post::Windows::Priv
+  include Msf::Post::Windows::Process
+
+  def initialize(info={})
+    super(update_info(info, {
+      'Name'          => 'Microsoft Windows Server 2003 SP2 Arbitrary Write Privilege Escalation',
```

Comment:
The module name could be a bit more meaningful. We try to not have target information included in them. Maybe something like ```tcpip.sys Arbitrary Write Privilege Escalation```